src/handlers/time-trail-race/components/NextPlayer.tsx

```diff
@@ -1,4 +1,4 @@
-import '../../poster/components/types/ScrollAnimation.scss';
+import '../../poster-gewis/components/types/ScrollAnimation.scss';
```

Comment:
The `ScrollAnimation` should be a generic poster component. I did a quick search, but it seems to only be used by the `TimeTrailRaceScoreboard`, though there might be uses for posters as well.

Reply:
Okay I went to look why this exactly was being used, and then I lost myself in a rabbit hole. I am unsure why something that is only used in time-trial-race is placed under posters. And I am pretty sure the VerticalScroll also implements this but without the CSS. But I think that is all out of scope for this PR.

I have moved it back to the base